common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -37,6 +57,27 @@ class VerticalBlock(SequenceFields, XModuleFields, StudioEditableBlock, XmlParse
 
     show_in_read_only_mode = True
 
+    def get_completables_for_user(self, username):
+        """
+        Return a set of descendent blocks that are completable.
```

Comment:
This isn't an accurate description anymore - this is the set of descendent blocks that this vertical should mark as complete once the user views them. (i.e. it excludes completable blocks that are complete, and it excludes completable blocks that get marked complete via some other mechanism).